packages/fluentui/docs/src/examples/components/Input/Variations/InputExampleWrapperSlot.shorthand.tsx

```diff
@@ -2,7 +2,15 @@ import * as React from 'react';
 import { Grid, Input, Text } from '@fluentui/react-northstar';
 
 const InputExampleWrapperSlot = () => (
-  <Grid columns="1fr 2fr" styles={{ justifyItems: 'start', alignItems: 'center', gap: '10px' }}>
+  <Grid
+    styles={{
+      gridTemplateColumns: '1fr 2fr',
+      msGridColumns: '1fr 2fr',
+      justifyItems: 'start',
+      alignItems: 'center',
+      gap: '10px',
```

Comment:
We have there a `gap`, but `msGridColumns` doesn't reflect this. I suggest to use there `columns` or even `Flex` to simplify example

Reply:
You mean to change the display to `flex`? Is `gap` equivalent to `gridGap`?